services/movie.py

```diff
@@ -0,0 +1,34 @@
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> list[Movie]:
+    queryset = Movie.objects.all()
+    if genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids)
+    return queryset
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
```

Comment:
The `get_movie_by_id` function uses `Movie.objects.get(id=movie_id)`, which will raise a `Movie.DoesNotExist` exception if the movie with the given ID is not found. Consider handling this exception to provide a more user-friendly error message or behavior.